base/essentials.jl

```diff
@@ -382,8 +382,17 @@ default_access_order(a::GenericMemory{:atomic}) = :monotonic
 default_access_order(a::GenericMemoryRef{:not_atomic}) = :not_atomic
 default_access_order(a::GenericMemoryRef{:atomic}) = :monotonic
 
-getindex(A::GenericMemory, i::Int) = (@_noub_if_noinbounds_meta;
-    memoryrefget(memoryrefnew(memoryrefnew(A), i, @_boundscheck), default_access_order(A), false))
+# bootstrap version for Memory{Any}
```

Comment:
I don't really understand this comment. What is the commented code used for? It is identical to what used to be there except with `Memory{Any}`?

Reply:
yeah that was a mistake.